.github/workflows/capabilities.yml

```diff
@@ -0,0 +1,33 @@
+name: '@web3-storage/capabilities'
+env:
+  CI: true
+  FORCE_COLOR: 1
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'packages/capabilities/**'
+      - '.github/workflows/capabilities.yml'
+      - 'pnpm-lock.yaml'
+  pull_request:
+    paths:
+      - 'packages/capabilities/**'
+      - '.github/workflows/capabilities.yml'
+      - 'pnpm-lock.yaml'
+jobs:
```

Comment:
Can you please add the typecheck action, because it reports issues inline with code. I think other actions had them and they don't seem to appear anymore, not sure why they got removed, but I find them to be very helpful.

Here is an example of it
https://github.com/web3-storage/ucanto/blob/18b3fd17e8d7671dc76a238723cb8d524b29cba3/.github/workflows/transport.yml#L22-L48

P.S.: I know build will fail if there are type errors, but it just dumps them into output as opposed to printing them out in the code.

Reply:
good idea, did it